CRITICAL - 2023-12-15 11:29:37 --> Undefined variable $nomor
#0 C:\xampp\htdocs\pdln\app\Models\PjumModel.php(92): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 92)
#1 C:\xampp\htdocs\pdln\app\Controllers\Admin\Dashboard.php(1570): App\Models\PjumModel->cekidpjum('4')
#2 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\Dashboard->importbiaya('4')
#3 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Dashboard))
#4 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-12-15 12:53:46 --> 7 : Failed to connect to www.bi.go.id port 443: Connection refused
#0 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(656): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('7', 'Failed to conne...')
#1 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(364): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(140): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://www.bi....')
#3 C:\xampp\htdocs\pdln\app\Controllers\Admin\PJUM.php(1247): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://www.bi....', Array)
#4 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\PJUM->tanggalpjum('11', '4', 'pjum', '', '')
#5 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\PJUM))
#6 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-15 13:36:40 --> Undefined variable $id_kurs
#0 C:\xampp\htdocs\pdln\app\Views\pb\v_datapb.php(288): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 288)
#1 C:\xampp\htdocs\pdln\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\pdln\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\pdln\system\Common.php(1089): CodeIgniter\View\View->render('pb/v_datapb', Array, true)
#4 C:\xampp\htdocs\pdln\app\Controllers\Admin\PB.php(500): view('pb/v_datapb', Array)
#5 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\PB->datapb('pb', '4', '8', '', '')
#6 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\PB))
#7 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-12-15 14:53:24 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\pdln\app\Controllers\Admin\Dashboard.php(987): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\Dashboard->importbiaya('4')
#2 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Dashboard))
#3 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-12-15 14:54:10 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\pdln\app\Controllers\Admin\Dashboard.php(987): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\Dashboard->importbiaya('4')
#2 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Dashboard))
#3 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-12-15 15:38:43 --> 7 : Failed to connect to www.bi.go.id port 443: Connection refused
#0 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(656): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('7', 'Failed to conne...')
#1 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(364): CodeIgniter\HTTP\CURLRequest->sendRequest(Array)
#2 C:\xampp\htdocs\pdln\system\HTTP\CURLRequest.php(140): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://www.bi....')
#3 C:\xampp\htdocs\pdln\app\Controllers\Admin\PB.php(974): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://www.bi....', Array)
#4 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\PB->editnopb('7', '4', 'pb', '', '')
#5 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\PB))
#6 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-15 16:03:42 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\pdln\app\Controllers\Admin\Transaksi.php(699): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 699)
#1 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\Transaksi->transaksi()
#2 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Transaksi))
#3 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-12-15 16:06:25 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\pdln\app\Controllers\Admin\Transaksi.php(699): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 699)
#1 C:\xampp\htdocs\pdln\system\CodeIgniter.php(832): App\Controllers\Admin\Transaksi->transaksi()
#2 C:\xampp\htdocs\pdln\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Transaksi))
#3 C:\xampp\htdocs\pdln\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pdln\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
